### ISSUE TITLE
Settings: Update ARIA labeling to use 'Modified in (workspace /user settings)

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#59492]
Make labeling match element text

**Comment by roblourens** [comment]
What text does it insert? In Insiders right now for me VoiceOver seems like it isn't reading the setting name.

**Comment by cleidigh** [comment]
something like:
Editor: Tab Size???breakpoints-test1:4

"breakpoints-test1" is the current branch name??? voiceover seems to have a wild pointer to the accessibility name.  Apple will be the death of me, had to upgrade everything just to get Xcode just to get the accessibility Debugger
now I can no longer toggle voiceover
perhaps tomorrow will be better... I have to get the two readers to work correctly, now it's a battle challenge  
sorry it's so problematic

**Comment by roblourens** [comment]
Very weird! I don't even get that.

**Comment by cleidigh** [comment]
Apples Mojave is toasting me.  changed bunch of security now you cannot do many things with Remote Desktop , this is insane I can no longer build and test on my Mac laptop.

**Comment by cleidigh** [comment]
finally building, making some progress

I think I have finally found some voiceover oddities with labeling.  It appears that voiceover within electron
does not like aria-labeledby pointing to a <span> this was confounding me for the element titles.
a quick experiment changing the element label from span to a div looks like it works.  any reason we can't do this using the span ? 

```ts

	private renderCommonTemplate(tree: ITree, container: HTMLElement, typeClass: string): ISettingItemTemplate {
		DOM.addClass(container, 'setting-item');
		DOM.addClass(container, 'setting-item-' + typeClass);
		const titleElement = DOM.append(container, $('.setting-item-title'));
		const labelCategoryContainer = DOM.append(titleElement, $('.setting-item-cat-label-container'));
		const categoryElement = DOM.append(labelCategoryContainer, $('span.setting-item-category'));
		const labelElement = DOM.append(labelCategoryContainer, $('setting-item-label'));
		const otherOverridesElement = DOM.append(titleElement, $('span.setting-item-overrides'));
```

**Comment by roblourens** [comment]
It won't show up yet because those aren't hooked up for folder settings, but if you are in a workspace with multiple folders, then you have 3 scopes - user, workspace, and folder - and that label can show that it's modified in the other 2.

**Comment by cleidigh** [comment]
so visually:
(also Modified in user, folder)

and these would be in separate spans?

**Comment by roblourens** [comment]
Correct

**Comment by cleidigh** [comment]
textContent will cross the spans so I think that's okay.
I had to do a lot of back-and-forth to satisfy both NVDA and voiceover, a lot of good learning that I should probably somehow pass on.  Maybe you know the best way to do that.

Regardless as small as it is in some respects, I think your new settings probably has some pretty good ARIA support.  At least until we that it doesn't !

**Comment by roblourens** [comment]
> I had to do a lot of back-and-forth to satisfy both NVDA and voiceover, a lot of good learning that I should probably somehow pass on. Maybe you know the best way to do that.

I think that either code comments describing how it works, or just leave your notes in this issue that's referenced from the commit message, whatever seems appropriate.
